modelina-website/.eslintignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-.next
\ No newline at end of file
+.next
+/**/*.css
```

Comment:
Why do we need this?

Reply:
Unfortunately, ts loader cannot do something unless it's typescript format, unless we migrate to swc or babel it can be included on eslint

Reply:
But we don't have any CSS files in our codebase 🤔